crypto/kyber/kyber512_ref.c

```diff
@@ -0,0 +1,31 @@
+/*
+------------------------------------------------------------------------------------
+ Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0 OR ISC
+------------------------------------------------------------------------------------
+*/
+
+#define KYBER_K 2
+
+#include "./pqcrystals_kyber_ref_common/params.h"
+#include "./pqcrystals_kyber_ref_common/cbd.h"
+#include "./pqcrystals_kyber_ref_common/fips202.h"
+#include "./pqcrystals_kyber_ref_common/indcpa.h"
+#include "./pqcrystals_kyber_ref_common/kem.h"
+#include "./pqcrystals_kyber_ref_common/ntt.h"
+#include "./pqcrystals_kyber_ref_common/poly.h"
+#include "./pqcrystals_kyber_ref_common/polyvec.h"
+#include "./pqcrystals_kyber_ref_common/reduce.h"
+#include "./pqcrystals_kyber_ref_common/symmetric.h"
+#include "./pqcrystals_kyber_ref_common/verify.h"
```

Comment:
Do we need to include all the header files? Won't each c file include what it needs?

Reply:
you are right, I removed the unnecessary includes.